apps/ledger-live-desktop/src/renderer/modals/Send/fields/RecipientField.react.test.tsx

```diff
@@ -1,19 +1,18 @@
 import React from "react";
+import axios from "axios";
 import "@testing-library/jest-dom";
 import BigNumber from "bignumber.js";
 import {
-  findCryptoCurrencyById,
+  getCryptoCurrencyById,
   setSupportedCurrencies,
 } from "@ledgerhq/live-common/currencies/index";
 import { Account } from "@ledgerhq/types-live";
 import { InvalidAddress } from "@ledgerhq/errors";
 import { ThemeProvider } from "styled-components";
-import network from "@ledgerhq/live-common/network";
```

Comment:
Yup, it's in a test file where I need to mock the API return, so I had to import Axios as dev dep and mock the lib to avoid calling the real backend.